src/style/list/conversations.less

```
@@ -17,54 +17,69 @@
  *
  */
 
+.conversations {
+  width: 320px;
+  background-color: @white;
```

```suggestion
  background-color: var(--app-bg-secondary);
```
that's white in light mode, black in dark mode if it's what design is going for